### PR TITLE
Bump up godep of etcd to use V3 API

### PR DESCRIPTION
**Pull request by hongchaodeng**
This is to separate the godep update commits of #22604

### CODE REVIEW

**Comment by wojtek-t** [comment]
@hongchaodeng - hmm why you are removing Godeps/LICENSE?
Can you remove the second commit completely?

**Comment by wojtek-t** [comment]
It seems that something is clearly wrong - can you please take a look?

**Comment by hongchaodeng** [comment]
The failure was
```
FAILED   hack/../hack/verify-generated-swagger-docs.sh
```
which I can't reproduce locally.

All I did is `rm -rf Godeps/ && godep save ./... && ./hack/update-godep-license.sh`

/cc @eparis @ixdy

**Comment by ixdy** [comment]
@hongchaodeng you can probably get it to repro locally if you update godep to v59. See #23200.

**Comment by hongchaodeng** [comment]
@ixdy 
I think the docs said k8s only worked with v53 or under. See
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md#installing-godep

So I keep using v53.

**Comment by hongchaodeng** [comment]
Alright I see. The submit queue was entirely blocked due to the godep issue :)
Will wait for the fix and rebase-push again.

/cc @wojtek-t @xiang90

**Comment by ixdy** [comment]
@k8s-bot unit test this please, issue #23200

**Comment by wojtek-t** [comment]
Seems to work now - thanks!

LGTM

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.5",
+	"GoVersion": "go1.4",
```
Comment (hongchaodeng) [inline Godeps/Godeps.json]: I just found that I shouldn't change the GoVersion.. Will fix it.

**Comment by hongchaodeng** [comment]
@wojtek-t 
Sorry for the hassle.
I undo the "GoVersion" change

**Comment by wojtek-t** [comment]
@hongchaodeng - thanks for catching it!

LGTM

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please github issue: #21484

**Pull request merged**